README.md

```diff
@@ -140,31 +140,35 @@ Learn more about how we review pull requests [here](docs/review-process.md).
 
 ![POAP Logo](public/images/poap-logo.svg)
 
-## Claim your POAP!
+## Claim your POAP and OATs!
 
-### What is POAP?
+### What is a POAP?
```

Comment:
Small nit-pick, but this changes it more to referencing the token itself ("a POAP"), but then the description calls it a "**dapp** that distributes badges in the form of ERC-721 tokens". Would consider just leaving `What is POAP?`, but not a hill worth dying on =)